### PR TITLE
added-notebook-files

### PR DESCRIPTION
**Pull request by murtazshaikh**
review pr...

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -0,0 +1,128 @@
+*{
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+}
+body{
+    min-height: 100vh;
```
Comment (syedabdullahali) [inline style.css:7]: Use group similar property

File: style.css [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+.paper-content{
+    position: absolute;
+    top: 30px;
+    right: 0px;
+    bottom: 30px;
+    left: 50px;
+    background: linear-gradient(transparent, transparent 28px, #91d1d3 28px);
+    background-size: 30px 30px;
+}
+.paper-content textarea{
```
Comment (syedabdullahali) [inline style.css:118]: here too

**Comment by syedabdullahali** [comment]
Embed `@mediaquery` at max-width: 400px; make it more responsive

File: style.css [hunk 3]
```diff
@@ -0,0 +1,128 @@
...
+    display: flex;
+    justify-content: center;
+    align-items: center;
+}
+.container{
+    position: absolute;
+    left: 50%;
+    transform: translateX(-50%);
+    perspective: 2000px;
+    perspective-origin: 50% 50%;
```
Comment (Abbas4tech) [inline style.css:22]: What does this property do ? @what does this property do @murtazshaikh

File: style.css [hunk 4]
```diff
@@ -0,0 +1,128 @@
...
+}
+.cover{
+    position: absolute;
+    min-width: 400px;
+    max-width: 800px;
+    height: 480px;
+    margin: 0 auto;
+    background-image: url(img/cover.jpg);
+    border-radius: 10px;
+    box-shadow: 0px 3px 10px rgba(0, 0, 0, 0.5);
```
Comment (Abbas4tech) [inline style.css:35]: How does this `10px` affect the shadow ?? @murtazshaikh

File: style.css [hunk 5]
```diff
@@ -0,0 +1,128 @@
...
+}
+.cover::after{
+    content: '';
+    position: absolute;
+    top: 0;
+    left: -20px;
+    bottom: 0;
+    width: 60px;
+    background: linear-gradient(transparent,transparent 13px,black 14px,black 16px,transparent 17px,transparent 30px) repeat-y;
+    box-sizing: border-box;
```
Comment (Abbas4tech) [inline style.css]: You've already reset to `box-sizing: border-box` why you use this twice ??? @murtazshaikh

File: style.css [hunk 6]
```diff
@@ -0,0 +1,128 @@
...
+    content: '';
+    position: absolute;
+    top: 0;
+    left: 0px;
+    bottom: 0;
+    width: 40px;
+    background: radial-gradient(#575450 6px, transparent 7px) repeat-y;
+    background-size: 30px 30px;
+    border-right: 3px solid #d44147;
+    box-sizing: border-box;
```
Comment (Abbas4tech) [inline style.css]: @murtazshaikh  same here ??? Can you explain or can you remove this ???

File: style.css [hunk 7]
```diff
@@ -0,0 +1,128 @@
...
+    width: 100%;
+    max-width: 100%;
+    height: 100%;
+    max-height: 100%;
+    line-height: 30px;
+    background: transparent;
+    padding: 0 10px;
+    border: 0;
+    resize: none;
+    outline: none;
```
Comment (Abbas4tech) [inline style.css:131]: What does `resize` and `outline` do ??
@murtazshaikh

**Review by Abbas4tech — CHANGES_REQUESTED** [review]

**Comment by Abbas4tech** [comment]
Group Similar Properties in Style Sheets @murtazshaikh

File: style.css (hunk 3 above)
Comment (murtazshaikh) [reply style.css:22]: It determines the position from where the viewer is looking, in this case 50% 50% means we are looking from center.

File: style.css (hunk 4 above)
Comment (murtazshaikh) [reply style.css:35]: 10px is of blur. More the pixel more the blur shadow will.

File: style.css (hunk 5 above)
Comment (murtazshaikh) [reply style.css]: My mistake, I have removed it.

File: style.css (hunk 6 above)
Comment (murtazshaikh) [reply style.css]: Removed

File: style.css (hunk 7 above)
Comment (murtazshaikh) [reply style.css:131]: resize :- In <textarea> by default we can resize the text-area, so to maker it non-resizable `resize: none;'  property is used.
outline :- whenever we start typing outline comes, so to remove it we use `outline: none' property.

**Comment by murtazshaikh** [comment]
> Group Similar Properties in Style Sheets @murtazshaikh

Done

**Comment by murtazshaikh** [comment]
> Embed `@mediaquery` at max-width: 400px; make it more responsive

Done @syedabdullahali

**Review by mujeeb100 — APPROVED** [review]

**Review by mmusk — APPROVED** [review]
LGTM

**Review by syedabdullahali — APPROVED** [review]

**Review by syedabdullahali — APPROVED** [review]

**Pull request merged**